### ISSUE TITLE
Поиск вакансий по городам работает с ошибкой

### ISSUE THREAD

**Issue by SibirBear** [Hexlet/hexlet-cv#318]
При поиске вакансий не выдает результаты по указанному городу, при том, что в списке без фильтров эти вакансии есть.
Москва, Санкт-Петербург, Минск - данные о вакансиях выводятся (на полноту не проверял).
Новосибирск, Ульновск - список пуст, хотя вакансии есть.

Если из самих вакансий переходить по ссылке города (место работы), то список вакансий по городу формируется корректно.
![image](https://user-images.githubusercontent.com/62481984/154016886-f1dc35f2-38d5-44d4-9f9b-d31155e85f23.png)

Примеры:
<details>
  <summary>Москва</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154014967-5aa80fe3-97d8-46b8-9967-b995738eb5cb.png)

</details>
<details>
  <summary>Санкт-Петербург</summary>
 
 _"петербург" с маленькой буквы устанавливается автоматически при нажатии на кнопку "Найти"_ 

![image](https://user-images.githubusercontent.com/62481984/154015655-34c8523b-d49b-4d84-a4c0-7b38f41c0737.png)

</details>
<details>
  <summary>Минск</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154015827-c73311c6-3a23-413a-96cd-45536f13cd8a.png)

</details>
<details>
  <summary>Новосибирск</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154015963-51528d37-14da-46ed-ab1a-b1316ecb61dd.png)

</details>
<details>
  <summary>Новосибирск (вакансия)</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154016156-b9ad602f-90e1-430a-b014-5996927f75ce.png)

</details>
</details>
<details>
  <summary>Новосибирск (переход к списку вакансий из карточки вакансии)</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154016440-78565ecd-9977-4780-a6ed-fed84777ef27.png)

</details>
</details>
<details>
  <summary>Ульяновск</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154016548-e610358d-a247-42a1-9a18-9dd9a44c2f7b.png)

</details>
<details>
  <summary>Ульяновск (вакансия)</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154016640-ead651d9-eebd-4a0e-bb99-8f3180495c0a.png)

</details>
<details>
  <summary>Ульяновск (переход к списку вакансий из карточки вакансии)</summary>
  
![image](https://user-images.githubusercontent.com/62481984/154016683-e87b93b0-aa6a-4809-a601-3cdea23cd06e.png)

</details>

**Comment by seth2810** [comment]
@fey провалидируй?

**Comment by fey** [comment]
Что-то тут не так. Без пробела вакансии все равно не выводятся

**Comment by seth2810** [comment]
Можешь уточнить, это уже после деплоя фикса? Сейчас вижу, что концевые пробелы в названиях городов не устранились - https://cv.hexlet.io/vacancies/390. А это может значить либо что фикс не раскатали, либо миграция не сработала.

**Comment by fey** [comment]
Миграция не сработала. Попробую пофиксить другой миграцией :D

**Comment by seth2810** [comment]
Да, уже заметил связанный issue #322. Если честно пока выглядит странно, учитывая что поле обязательное.

**Comment by fey** [comment]
Раньше как работало - можно было создать вакансию без зп от\до, чтобы отобразить её с "зп по договоренности". Потом добавили валидацию, что зп обязательная (в модели), а старые записи остались.

**Comment by seth2810** [comment]
@fey поиск заработал. Считаю, что issue можно закрывать.
